lib/metacomponents/Flexboxable.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'react-emotion'
+import {camelCaseToPipeCase} from '~/utils/naming'
+
+const Flexboxable = Component =>
+  styled(Component)`
+    ${props =>
+      props.alignContent &&
+      `align-content: ${camelCaseToPipeCase(props.alignContent)}`};
```

Comment:
The extra semi-colons on the outside will still be output if the prop isn't present, which is a bit awkward.